src/alfasim_score/converter/alfacase/score_input_reader.py

```diff
@@ -312,3 +313,45 @@ def read_operation_fluid_data(self) -> Dict[str, Any]:
             "api_gravity": Scalar(fluid_data["api_gravity"], FRACTION_UNIT),
             "gas_gravity": Scalar(fluid_data["gas_gravity"], FRACTION_UNIT),
         }
+
+    def read_output_curves(self) -> Dict[str, Array]:
+        """
+        Get the results from the input file.
+        This parsed output curves are used to check cases results.
+        """
+        if "result" in self.input_content["operation"]["thermal_simulation"]:
```

Comment:
A more "fancie" approach could be:
```
if results := self.input_content["operation"]["thermal_simulation"].get("result"):
    result = results["0"]
```

Reply:
Honestly, I think this sintax is terrible hahahhaha.